backend/packageLibrary.go

```diff
@@ -23,10 +26,18 @@ func RetrieveAllApps(beta bool) ([]data.PackageInfo, error) {
 	var tmpPreview data.PackageInfo
 	// step: iterate on packages
 	for _, pkg := range storeItems {
-		// skip if ignore beta app
-		// if !beta && pkg.Beta {
-		// 	continue
-		// }
+		if beta && pkg.Beta {
+			if pkg.Id == "filebrowser" {
```

Comment:
@bonhokage06 you dont need to check specific package. As long as it is marked of Beta and included as test user then it will return the package